actionpack/lib/action_dispatch/http/mime_negotiation.rb

```diff
@@ -60,7 +60,7 @@ def formats
         fetch_header("action_dispatch.request.formats") do |k|
           params_readable = begin
                               parameters[:format]
-                            rescue ActionController::BadRequest
+                            rescue ActionController::BadRequest, Parameters::ParseError
```

Comment:
Maybe we should put it somewhere in here instead?

https://github.com/rails/rails/blob/98c1432583d4607af1467425183d8d448a692c5a/actionpack/lib/action_dispatch/http/parameters.rb#L54-L58

Reply:
@gmcgibbon you knows it better than me. Feel free to adjust the PR.